checkpoint-single-nic/azuredeploy.json

```diff
@@ -0,0 +1,305 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminPassword": {
+            "metadata": {
+                "description": "Administrator password"
+            },
+            "type": "securestring"
+        },
+        "authenticationType": {
+            "allowedValues": [
+                "password",
+                "sshPublicKey"
+            ],
+            "defaultValue": "password",
+            "metadata": {
+                "description": "Authentication type"
+            },
+            "type": "string"
+        },
+        "newStorageAccountName": {
+            "metadata": {
+                "description": "Name of storage account"
+            },
+            "type": "string"
+        },
+        "offer": {
+            "allowedValues": [
+                "R7730BYOL",
+                "R7730PAYG"
+            ],
+            "defaultValue": "R7730BYOL",
+            "metadata": {
+                "description": "Offer version and type"
+            },
+            "type": "string"
+        },
+        "sshPublicKey": {
+            "defaultValue": "",
+            "metadata": {
+                "description": "Administrator SSH public key"
+            },
+            "type": "string"
+        },
+        "vmName": {
+            "metadata": {
+                "description": "Name of the Security Gateway"
+            },
+            "type": "string"
+        },
+        "vmSize": {
+            "defaultValue": "Standard_D3",
+            "metadata": {
+                "description": "Size of the VM"
+            },
+            "type": "string"
+        }
+    },
+    "variables": {
+        "apiVersion": "2015-06-15",
+        "customData": "#!/bin/bash",
+        "addressPrefix": "10.0.0.0/16",
+        "gatewayPrivateAddress": "10.0.1.10",
+        "onPremiseNetwork": "198.51.100.0/24",
```

Comment:
This is just a placeholder, and the customer could replace it with his own.
Note that 198.51.100.0/24 is part of an address range reserved for documentation.
See: https://tools.ietf.org/html/rfc5737